src/hugo/content/en-us/implementations/aws/command/command2/_index.md

```diff
@@ -0,0 +1,721 @@
+- _ShadowTopicPrefix_ is an abbreviated form of the topic, referred as `$aws/things/device1/shadow` (classic shadow)
+
+1. The _Device_ establishes an MQTT connection to the _AWS IoT Core_ endpoint and then subscribes to the reserved [MQTT shadow topics](https://docs.aws.amazon.com/iot/latest/developerguide/reserved-topics.html#reserved-topics-shadow) to get and update shadow event operations. Moreover, instead of subscribing individually to each reserved shadow topic, wildcard(_#_) could be used for generalized subscription like _`ShadowTopicPrefix/get/#`_ , _`ShadowTopicPrefix/update/#`_.
+
+    - `ShadowTopicPrefix/get/accepted`
+    - `ShadowTopicPrefix/get/rejected`
+    - `ShadowTopicPrefix/update/accepted`
+    - `ShadowTopicPrefix/update/rejected`
+    - `ShadowTopicPrefix/update/delta`
+
+2. After successful connect and topic subscription, the _Device_ publish request to _`ShadowTopicPrefix/get`_ topic and process the latest shadow file received on the _`ShadowTopicPrefix/get/accepted`_ topic.
+3. After receiving  the _command_ on shadow file,  the _Device_ makes publish request to `cmd/device1/response` topic as a command execution response and on completion of command,  the _Device_  publish status on `ShadowTopicPrefix/update` topic for reconciliation of _reported_ state on shadow file.     
+4. Once the delta attributes are processed from the device shadow get request on device initial connect/reconnect, and if device optionally remain connected, then it can further receive delta changes on shadow file from the _Client Application_.  
+5. The _Client Application_ establishes an MQTT connection to the _AWS IoT Core_ endpoint, and then subscribe to the `ShadowTopicPrefix/update/accepted`,  `ShadowTopicPrefix/update/rejected` shadow topics to reconcile shadow file updates made by device. The _Client Application_ also subscribes to the `cmd/device1/response` topic to process command execution response.
+6. To send a command, the _Client Application_ publishes a message on the `ShadowTopicPrefix/update` topic by adding command request under _desired_ state data for device. 
+
+
+
+```plantuml
+@startuml
+!define AWSPuml https://raw.githubusercontent.com/awslabs/aws-icons-for-plantuml/v7.0/dist
+!includeurl AWSPuml/AWSCommon.puml
+!includeurl AWSPuml/InternetOfThings/all.puml
+!includeurl AWSPuml/General/Client.puml
+
+'Comment out to use default PlantUML sequence formatting
+skinparam participant {
+    BackgroundColor AWS_BG_COLOR
+    BorderColor AWS_BORDER_COLOR
+}
+'Hide the bottom boxes
+hide footbox
+
+participant "<$IoTGeneric>\nDevice" as device
+participant "<$IoTShadow>\nShadow" as shadow
+participant "<$IoTCore>\nMQTT Broker" as broker
+participant "<$Client>\nApplication" as app
+
+== Device connect and subscribe ==
+device -> broker : connect(iot_endpoint)
+device -> broker : subscribe("$aws/things/device1/shadow/get/accepted")
```

Comment:
As with other sequence diagram, maybe combine to a single wildcard subscription.

Reply:
I added for additional clarity, let me know if explicit mention will cause any issues.